hyacinth/session.py

```diff
@@ -85,11 +115,11 @@ def __init__(
         self.ratelimit_remaining = math.inf
         self.raise_for_status = raise_for_status
 
-    @staticmethod
-    def __make_url(path):
-        return f"{CLIO_API_BASE_URL_US}/{path}.json"
+    def make_url(self, path):
```

Comment:
Curious why you made this change?

Reply:
This one was to make it easier to test the changes. I also thought to myself "Why make this private? 🤔 "